### PR TITLE
Use MessageText in SystemMessage

### PR DESCRIPTION
**Pull request by KDederichs**
As outlined in https://github.com/FaridSafi/react-native-gifted-chat/issues/1645 it would be nice to use parsed messages in SystemMessages

### CODE REVIEW

**Comment by KDederichs** [comment]
also not stale

**Comment by KDederichs** [comment]
still not stale

File: src/TypingIndicator.tsx [hunk 1]
```diff
@@ -8,9 +9,9 @@ interface Props {
 }
 
 const TypingIndicator = (props: Props) => {
-  const [yCoords, setYCoords] = useState(new Animated.Value(200))
-  const [heightScale, setHeightScale] = useState(new Animated.Value(0))
-  const [marginScale, setmarginScale] = useState(new Animated.Value(0))
+  const [yCoords] = useState(new Animated.Value(200))
```
Comment (xcarpentier) [inline src/TypingIndicator.tsx:12]: Why?

Comment (KDederichs) [reply src/TypingIndicator.tsx:12]: It's unused, why keep it and let the TS compiler complain about unused values?

**Pull request closed without merging**